scripts/us_census/geo/census_divisions.py

```diff
@@ -0,0 +1,167 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#         https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+'''
+Generate MCF nodes for US Census Divisions and Regions
+
+To run, download the codes from
+https://www2.census.gov/programs-surveys/popest/geographies/2018/state-geocodes-v2018.xlsx
```

Comment:
For my understanding how did you come across this? Is it referenced somewhere in census website?

Reply:
I just did a search for [Census divisions FIPS] and picked the latest xls. I didn't see a specific link in the census docs.